app/models/user.rb

```diff
@@ -33,9 +35,22 @@ def self.from_omniauth(auth)
   def self.new_with_session(params, session)
     super.tap do |user|
       if data = session["devise.github_data"] &&
-         session["devise.github_data"]["extra"]["raw_info"]
+                session["devise.github_data"]["extra"]["raw_info"]
         user.email = data["email"] if user.email.blank?
       end
     end
   end
+
+  def self.create_default_user!
+    User.create! email: User::DEFAULT_USER_EMAIL, password: '12345678'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.